examples/kubernetes/connectivity-check/Makefile

```diff
@@ -92,10 +92,13 @@ help:
 list:
 	$(QUIET)$(CUE) cmd ls
 
+fmt:
+	$(QUIET)$(CUE) fmt
```

Comment:
If I am not wrong then this only formats the cue templates and not check for formatting errors, right?

Reply:
Ah, I missed the `git diff` part. Sorry
Looks good.